### ISSUE TITLE
Tested kerberos tutorial, but no successful connection with inbo-sql07-prd.inbo.be

### ISSUE THREAD

**Issue by florisvdh** [inbo/tutorials#41]
I followed the installation tutorial. I get a ticket, but I get an error with the `sqlcmd` command. I did the test while being connected to the wireless INBO network. This is my output (I masked the username):
```
$ kinit my_username
Password for my_username@INBO.BE: 

$ klist
Ticket cache: FILE:/tmp/krb5cc_1000
Default principal: my_username@INBO.BE

Valid starting     Expires            Service principal
08-02-18 17:22:55  09-02-18 03:22:55  krbtgt/INBO.BE@INBO.BE
	renew until 15-02-18 17:22:49

$ sqlcmd -S inbo-sql07-prd.inbo.be -E
Sqlcmd: Error: Microsoft ODBC Driver 13 for SQL Server : SSPI Provider: Cannot find KDC for realm "INBO.BE".
Sqlcmd: Error: Microsoft ODBC Driver 13 for SQL Server : Cannot generate SSPI context.
```

Any hints? Network reasons? May have to do with account permissions. I saw something similar on [SO](https://stackoverflow.com/questions/46330091/sql-server-odbc-connection-with-kerberos-ad-authentication).

**Comment by Jo-Loos** [comment]
reason is the difference in dns server seection in linux and windows (&
mac)

we 're working on a solution that doesnt break other functionality

mvg
jol



Op 10 feb. 2018 11:51 a.m. schreef "Floris Vanderhaeghe" <
notifications@github.com>:

I followed the installation tutorial. I get a ticket, but I get an error
with the sqlcmd command. I did the test while being connected to the
wireless INBO network. This is my output (I masked the username):

$ kinit my_username
Password for my_username@INBO.BE:

$ klist
Ticket cache: FILE:/tmp/krb5cc_1000
Default principal: my_username@INBO.BE

Valid starting     Expires            Service principal
08-02-18 17:22:55  09-02-18 03:22:55  krbtgt/INBO.BE@INBO.BE
	renew until 15-02-18 17:22:49

$ sqlcmd -S inbo-sql07-prd.inbo.be -E
Sqlcmd: Error: Microsoft ODBC Driver 13 for SQL Server : SSPI
Provider: Cannot find KDC for realm "INBO.BE".
Sqlcmd: Error: Microsoft ODBC Driver 13 for SQL Server : Cannot
generate SSPI context.

Any hints? Network reasons? May have to do with account permissions. I saw
something similar on SO
<https://stackoverflow.com/questions/46330091/sql-server-odbc-connection-with-kerberos-ad-authentication>
.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/inbo/tutorials/issues/41>, or mute the thread
